packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -90,6 +90,7 @@ const TSErrors = makeErrorTemplates(
     EmptyTypeParameters: "Type parameter list cannot be empty.",
     ExpectedAmbientAfterExportDeclare:
       "'export declare' must be followed by an ambient declaration.",
+    IdentifierExpected: "Identifier expected.",
```

Comment:
Can we provide a more specific error message?

Reply:
> Expected: Babel should throw an identifier is expected after interface, aligning to TSC behaviour: TS Playground

Reply:
Do we need to follow the description from `Expected` section?

Reply:
Aligning to `tsc` doesn't always mean to throw the same errors😄
Probably `'export default interface' must be followed by an identifier.` would be better.

Reply:
> Babel should throw an identifier is expected after interface
It seems the issue description should be changed to avoid confusion.

Reply:
We are not handling interfaces here but interfaces exported as default. We can provide a specific error message that would be more useful to users.